test/spatial_reaction_systems/lattice_reaction_systems_jumps.jl

```diff
@@ -0,0 +1,160 @@
+### Preparations ###
+
+# Fetch packages.
+using JumpProcesses
+using Random, Statistics, SparseArrays, Test
+
+# Fetch test networks.
+include("../spatial_test_networks.jl")
+
+### Correctness Tests ###
```

Comment:
There is no correctness being checked. Only the retcode of the solution.

Reply:
We should definitely test correctness. You can take the ABC test from JumpProcesses or the any other spatial test.

Reply:
Maybe I should rename that header. We do perform more tests than just checking the output. E.g. the ABC model from JumpProcesses have been converted to Catalyst and is tested later in the file.